deployment/maintain/rebase-helper.md

```diff
@@ -52,34 +55,34 @@ Rebase-helper provides an API which you can use either directly from Python, or
 Example of patching new sources and starting scratch builds with fedpkg.
  This returns task_ids. The bash equivalents are included for comparison:
  
-* ``Python API``
+* **Python API**
 
-```
-from rebasehelper.application import Application
-cli = CLI([‘--non-interactive, ‘--builds-nowait’, ‘-buildtool’, ‘fedpkg’, upstream_version])
-rh = Application(cli)
-rh.set_upstream_monitoring() # Switch rebase-helper to upstream release monitoring mode.
-rh.run()
-```
+   ```python3
+   from rebasehelper.application import Application
+   cli = CLI(['--non-interactive', '--builds-nowait', '--buildtool', 'fedpkg', 'upstream_version'])
+   rh = Application(cli)
+   rh.set_upstream_monitoring() # Switch rebase-helper to upstream release monitoring mode.
+   rh.run()
+   ```
 
-* ``BASH``
+* **BASH**
 
-```
-rebase-helper --non-interactive --builds-nowait --buildtool fedpkg upstream_version
-```
+   ```sh
+   rebase-helper --non-interactive --builds-nowait --buildtool fedpkg upstream_version
+   ```
 
-### Download logs and RPMs and compare with tools like abipkgdiff
+### Download logs and RPMs and compare with tools like ``abipkgdiff``
 
-* ``Python API``
+* **Python API**
```

Comment:
It looks like there's a lot of directions in one page. Would you mind splitting it into multiple pages?

Reply:
We've kind of moved beyond "directions" by this point in the document, TBH. 

Everything from "Integrating rebase-helper with an upstream monitoring service" down is a separate topic. I think it's fine to be its own separate page, but it's also not really "directions". It's more a rough sketch of possible pseudocode that might be employed for that purpose. 

You asked elsewhere if the code has been tested on current Fedora. As far as I know, this code _never_ worked on **any** version of Fedora / fedpkg. I had nothing to do with drafting the original document, but my impression is that whoever wrote it just jotted down some half-formed thoughts and never really fleshed them out. One of my original commits included this note:

> The syntax highlighting was also very useful as it highlighted several typos in the API examples. This fixes some more of them, as best I could _guess_ what they were supposed to be. But there are still some issues, since (for example) `fedpkg` isn't a legal argument to `--buildtool`. (The F28 `rebase-helper` says: "`ERROR: argument --buildtool: invalid choice: 'fedpkg' (choose from 'copr', 'koji', 'mock', 'rpmbuild')`".) But I only corrected what I could be reasonably sure of.

I would be very surprised if any of the code in these examples was 100% usable as-is. But, while I can correct the formatting, I can't correct the code. I didn't write it.

I'll part out the "Integrating..." bit into its own page, though.

Reply:
Thanks.
So fixing the code should be on issues list.
Every code snippet on this web should work (at least at some point), otherwise it's useless.